client/src/app/app.component.ts

```diff
@@ -6,6 +6,7 @@ import { Component, OnInit } from '@angular/core';
     <div>
         <app-header></app-header>
         <router-outlet></router-outlet>
+        <app-header></app-header>
```

Comment:
<app-header> po co? ;)

Reply:
kurde zapomniałem usunąć, to taka makieta footera :D żeby mniej więcej zobrazować sobie całość